setupKAppNavLib.sh

```diff
@@ -42,28 +69,26 @@ handleParams() {
 }
 
 build() {
-    echo "########## Build started on $(date)  ##########"
-    ./build.sh
+    echo "########## Build $reposArg started on $(date)  ##########"
```

Comment:
I think the `echo`s in build and push functions need to be updated with the new variables being used.